### ISSUE TITLE
Running the example code just throws an error

### ISSUE THREAD

**Issue by Mihai-B** [Maksimka101/combine#17]
Hi I am trying to use this plugin but when I run the example code, 

```
CombineInfo combineInfo = await Combine().spawn((context) {
  context.messenger
    ..messages.listen(
      (event) => print("Message from Main Isolate: $event"),
    )
    ..send("Hello from Combine Isolate!");
});

combineInfo.messenger.messages.listen(
  (event) {
    print("Message from Combine Isolate: $event");
    combineInfo.messenger.send("Hello from Main Isolate!");
  },
);

```
I get this exception:

```
I/flutter (10641): Message from Combine Isolate: Hello from Combine Isolate!
E/flutter (10641): [ERROR:flutter/runtime/dart_isolate.cc(1097)] Unhandled exception:
E/flutter (10641): UnimplementedError
E/flutter (10641): #0      IsolateBinding.platformDispatcher (package:combine/src/bindings/isolate_bindings/isolate_binding.dart:21:5)
E/flutter (10641): #1      ServicesBinding._initKeyboard.<anonymous closure> (package:flutter/src/services/binding.dart:73:7)
E/flutter (10641): <asynchronous suspension>
```

Flutter branch: Beta
Combine version:  0.5.4

What am I doing wrong?

**Comment by Maksimka101** [comment]
Hello

I can see an error in the example app. However my error is different
```
flutter: The following _TypeError was thrown during a platform message response callback:
flutter: type 'Null' is not a subtype of type 'List<dynamic>' in type cast
flutter:
flutter: When the exception was thrown, this was the stack:
flutter: #0      BackgroundIsolateBinaryMessenger.ensureInitialized.<anonymous closure>
_background_isolate_binary_messenger_io.dart:51
flutter: #7      _RawReceivePort._handleMessage (dart:isolate-patch/isolate_patch.dart:189:12)
flutter: (elided 6 frames from dart:async)
```

Can you please answer these questions? Or at least few of them

1. Have you changed the example app? 
2. What flutter version do you use?
3. What `combine` version do you use (version from pubspec.lock)?

**Comment by Mihai-B** [comment]
I noticed in commits that you just updated to the latest STABLE branch the Flutter SDK, I am however on the Beta branch of Flutter.

**Comment by Maksimka101** [comment]
Thank you. I'll deal with it soon

**Comment by Maksimka101** [comment]
Job done! This problem should be fixed in the combine 0.5.5

Please, let me know, whether it's fixed or not

**Comment by Mihai-B** [comment]
Hello, it looks like the issue is fixed with combine 0.5.5. Thank you for the very fast bug fix.
